pkg/controller/util/node/controller_utils.go

```diff
@@ -32,10 +32,10 @@ import (
 	"k8s.io/api/core/v1"
 	clientset "k8s.io/client-go/kubernetes"
 	appsv1listers "k8s.io/client-go/listers/apps/v1"
+	nodepkg "k8s.io/component-helpers/node/utils/node"
```

Comment:
Might as well standardize on `utilnode` while you're updating things.

Although if we're going to have some stuff in `k8s.io/kubernetes/pkg/util/node` and some in `k8s.io/component-helpers/node/utils/node`, then that complicates things since some files may need to include both...

`nodehelpers` would be consistent with some other component-helpers imports.

Although also, `component-helpers/node/utils/node` seems redundant... either the functions should just be under `component-helpers/node/utils` or else they should be split up into appropriately-named subdirectories (under either `component-helpers/node` or `component-helpers/node/utils`... This is probably more a question for sig-node-approvers since they're the OWNERS of that directory...

Reply:
okay @danwinship I will work on the required changes.

Reply:
@danwinship As discussed, I have made the appropriate changes suggested. Please review once. let me know if any changes required